logstash-commons/src/main/java/org/apache/mesos/logstash/common/network/NetworkUtils.java

```diff
@@ -0,0 +1,131 @@
+package org.apache.mesos.logstash.common.network;
+
+import org.apache.commons.exec.CommandLine;
+import org.apache.commons.exec.DefaultExecutor;
+import org.apache.commons.exec.PumpStreamHandler;
+import org.apache.commons.exec.environment.EnvironmentUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.log4j.Logger;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+import java.net.InterfaceAddress;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.net.UnknownHostException;
+import java.nio.charset.Charset;
+import java.util.Collections;
+import java.util.Enumeration;
+import java.util.Map;
+
+/**
+ * Utilities to help with networking
+ */
+@SuppressWarnings("PMD.AvoidUsingHardCodedIP")
+public class NetworkUtils {
```

Comment:
All other utils/services/etc are now Spring beans so please keep the style that way. At least for the sake of testing. Let me know if you need any help converting it.